src/main/java/io/jenkins/plugins/sample/AndrasPlugin.java

```diff
@@ -58,10 +65,20 @@ public boolean configure (StaplerRequest req, JSONObject formData) throws FormEx
         }
 
         public FormValidation doCheckGlobalName(@QueryParameter String globalName) {
+            Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```

Comment:
this prevents access to check the name, but anyone can call configure (there is no protection on that), is that correct?

Reply:
For the `configure` method, the permissions are checked in the class `Jenkins`, method `doConfigSubmit` (`checkPermission(MANAGE);`)
And I have checked, if not logged in, I cannot access http://localhost:8080/jenkins/configure